trainer/finetune.py

```diff
@@ -31,16 +31,33 @@
 flags.DEFINE_string("name", "default", "name/description of model  version")
 flags.DEFINE_enum("mode", "all", ["train", "evaluate", "all"],
                   "run mode: train, evaluate, or all")
+flags.DEFINE_enum("task", "rd_recommendations", ["rd_recommendations",
```

Comment:
Expand the doc a little but if needed, as this is quite important to what they do?

Reply:
where do you think this should go? in the flag description string or in the readme or something? I think the readme might be a good place.